dpgen/generator/run.py

```diff
@@ -2177,7 +2220,7 @@ def _read_model_devi_file(
         assert all(
             model_devi_content.shape[0] == model_devi_contents[0].shape[0]
             for model_devi_content in model_devi_contents
-        ), r"Not all beads generated the same number of lines in the model_devi${ibead}.out file. Check your pimd task carefully."
+        ), "Not all beads generated the same number of lines in the model_devi$\{ibead\}.out file. Check your pimd task carefully."
```

Comment:
This change should be reverted.

Reply:
This change is made by your bot, not me.

Reply:
It's made by https://github.com/deepmodeling/dpgen/commit/8f7f491bfa6d850d344267e0f3928aca6ae90b46